README.md

```diff
@@ -13,7 +13,7 @@
 
 ## About
 
-Imageing is a modern file sharing app for Android and iOS.
+imageing (and so are you!) is a modern file sharing app for Android and iOS.
```

Comment:
huh?

Reply:
cheeze said the funni yesterday, "I'm ageing" and I did on saturday

Reply:
acceptable